doc/guides/backport-to-lts-branches.md

```diff
@@ -0,0 +1,124 @@
+
+- Make sure up to date with staging
+  ```bash
+  git fetch --all
+  git checkout -B v8.x-staging DANGER/v8.x-staging
+  ```
+
+- Run the tests to make sure everything works:
+  ```bash
+  ./configure && make -j4 test
+  ```
+
+- Install branch-diff (and set up the key)
+  ```bash
+  npm install -g branch-diff
+  ```
+
+### Triage PRs
+
+Go through [PRs with the v8.x label](https://github.com/nodejs/node/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3Av8.x), for each PR:
+
+- Run CI
+- Check approvals (you can approve yourself)
+- Check SemVer
+- Check the PR metadata wasn't changed from `master` commit.
+- If there are merge conflicts, comment in the issue asking the PR author to rebase.
+- Land if okay
+  - Add git alias if you haven't already:
+  ```bash
+  git config --global alias.pa '!curl -L $1.patch | git am -3 -S --whitespace=fix #'
+  ```
+  - Patch Pull Request onto staging branch:
+  ```bash
+  git pa https://github.com/nodejs/node/pull/12345
+  ```
+  - If it doesn't apply cleanly then abort with:
+  ```bash
+  git am --abort
+  ```
+  - If it lands and tests pass, then add the `Backport-PR-URL:` line to each commit message:
+  ```bash
+  # If a single commit:
+  git commit --amend
+
+  # If multiple commits:
+  git rebase -i @~4 # If 3 commits.
+  # Then change pick to reword for each commit
+  #
+
+  # Add this line to each commit under the PR-URL:
+  # Backport-PR-URL: https://github.com/nodejs/node/pull/12345
+  ```
+  - Test the commit and push it
+    ```bash
+    ./configure && make -j4 test
+    git push
+    ```
+  - Close the backport PR with `Landed in $(git rev-parse --short HEAD)`, update the label on the original PR from `backport-requested-v8.x` to `backported-to-v8.x`.
+
+### Triage commits
+
+- Run branch diff /w parameters
+  ```bash
+  branch-diff ???
+  ```
+- For each PR
+ - Add `dont-land-on-v8.x` label if it shouldn't land.
+ - If you're not sure comment in the PR asking if it should land.
+ - If you think it's risky so should wait for a while, add the `baking-for-lts` tag.
+ - If it should land
+   - Cherry-pick the commits that landed on master:
+   ```bash
+   # If single commit:
+   git cherry-pick -S a03e27c
+   # If multiple commits:
+   git cherry-pick -S a03e27c 24360df 7fb455a
+   # If a range of commits:
+   git cherry-pick -S a03e27c...24360df
+   ```
+   - If there are simple conflicts resolve them, otherwise add the `backport-requested-v8.x` label and abort the cherry-pick.
+   ```bash
+   git cherry-pick --abort
+   ```
+   - Run the tests and make sure they pass:
+   ```bash
+   ./configure && make -j4 test
+
+   # If source wasn't changed (comments or docs):
+   make lint
+
+   # Run only tests changed in the last commit:
+   tools/test.py -J $(git show --name-only --pretty='' | grep 'test/')
+   ```
+   - Add `land-on-v8.x`, git push commits to `v8.x-staging`:
+   ```bash
+   git push
+   ```
+
+
+# How to backport to the current release
+
+- Run branch-diff to generate a list of shas:
```

Comment:
Nit: `shas` -> `SHAs` or even `SHA checksums` or whatever the best word (instead of `checksums`) might be there (`commit hashes`?).